lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -569,7 +614,11 @@ export class UserAgentApplication {
       this.cacheStorage.setItem(authorityKey, this.authority, this.inCookie);
 
       // Build the URL to navigate to in the popup window
-      const urlNavigate = authenticationRequest.createNavigateUrl(scopes)  + Constants.response_mode_fragment;
+      let urlNavigate = authenticationRequest.createNavigateUrl(scopes)  + Constants.response_mode_fragment;
```

Comment:
saw this a second time, could factor out

Reply:
Done. The new code is optimized.